server/auth/types/openid/routes.ts

```diff
@@ -188,7 +189,8 @@ export class OpenIdAuthRoutes {
           const user = await this.securityClient.authenticateWithHeader(
             request,
             this.openIdAuthConfig.authHeaderName as string,
-            `Bearer ${tokenResponse.idToken}`
```

Comment:
Do we need to do this for OpenID? If we do, do we need to add it to the refresh token flow? https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/auth/types/openid/openid_auth.ts#L303

Reply:
Do we want to add it for all the other auth types as well, such as proxy auth?

Reply:
It is a place holder, and adding it here should suffice. OpenId directly calls the openid url and doesn't rely on error response from backend, like SAML, to redirect to the IdP. Hence, instead of `undefined` I chose to add a placeholder.